notebooks/02_model/fm_deep_dive.ipynb

```diff
@@ -0,0 +1,1530 @@
+{
```

Comment:
<p>xlearn reminds me a lot to vw in the sense that you compute something and save to file, is there a way to compute things in memory so we can get rid of model_file for example?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
AFAIK there is no such API but we can definitely check with Chao